static/schemas/2.0/skus.2.0.json

```diff
@@ -0,0 +1,248 @@
+{
+    "title": "Skus.2.0",
+    "definitions": {
+        "SkusRules": {
```

Comment:
This is all the rules that list in the docs (http://rackhd.readthedocs.io/en/latest/rackhd2.0/skus.html?highlight=contains#sku-json-format), your code doesn't match with it